src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -84,9 +84,7 @@ CHIP_ERROR CASEServer::OnMessageReceived(Messaging::ExchangeContext * ec, const
         bool watchdogFired = GetSession().InvokeBackgroundWorkWatchdog();
         if (!watchdogFired)
         {
-            // Handshake wasn't stuck, let it continue its work
```

Comment:
This comment should stay, no?  It explains what situation this case is handling....